### ISSUE TITLE
Call to a member function id() on null

### ISSUE THREAD

**Issue by sinanatra** [Daniel-KM/Omeka-S-module-Annotate#5]
i correctly install the module, but this happens when i open an item:
``` 2021-10-05T14:24:10+00:00 ERR (3): Error: Call to a member function id() on null in .../omeka/modules/Annotate/Module.php:1037
Stack trace:
#0 .../omeka/modules/Annotate/Module.php(961): Annotate\Module->displayForm(Object(Laminas\EventManager\Event))
#1 .../omeka/vendor/laminas/laminas-eventmanager/src/EventManager.php(321): Annotate\Module->displayListAndForm(Object(Laminas\EventManager\Event))
#2 .../omeka/vendor/laminas/laminas-eventmanager/src/EventManager.php(170): Laminas\EventManager\EventManager->triggerListeners(Object(Laminas\EventManager\Event))
#3 .../omeka/application/src/View/Helper/Trigger.php(61): Laminas\EventManager\EventManager->triggerEvent(Object(Laminas\EventManager\Event))
#4 [internal function]: Omeka\View\Helper\Trigger->__invoke('view.show.after')
#5 .../omeka/vendor/laminas/laminas-view/src/Renderer/PhpRenderer.php(396): call_user_func_array(Object(Omeka\View\Helper\Trigger), Array)
#6 .../omeka/application/view/omeka/admin/item/show.phtml(98): Laminas\View\Renderer\PhpRenderer->__call('trigger', Array)
#7 .../omeka/vendor/laminas/laminas-view/src/Renderer/PhpRenderer.php(505): include('/Users/giacomon...')
#8 .../omeka/vendor/laminas/laminas-view/src/View.php(206): Laminas\View\Renderer\PhpRenderer->render(NULL)
#9 .../omeka/vendor/laminas/laminas-view/src/View.php(235): Laminas\View\View->render(Object(Laminas\View\Model\ViewModel))
#10 .../omeka/vendor/laminas/laminas-view/src/View.php(199): Laminas\View\View->renderChildren(Object(Laminas\View\Model\ViewModel))
#11 .../omeka/vendor/laminas/laminas-mvc/src/View/Http/DefaultRenderingStrategy.php(104): Laminas\View\View->render(Object(Laminas\View\Model\ViewModel))
#12 .../omeka/vendor/laminas/laminas-eventmanager/src/EventManager.php(321): Laminas\Mvc\View\Http\DefaultRenderingStrategy->render(Object(Laminas\Mvc\MvcEvent))
#13 .../omeka/vendor/laminas/laminas-eventmanager/src/EventManager.php(170): Laminas\EventManager\EventManager->triggerListeners(Object(Laminas\Mvc\MvcEvent))
#14 .../omeka/vendor/laminas/laminas-mvc/src/Application.php(366): Laminas\EventManager\EventManager->triggerEvent(Object(Laminas\Mvc\MvcEvent))
#15 .../omeka/vendor/laminas/laminas-mvc/src/Application.php(347): Laminas\Mvc\Application->completeRequest(Object(Laminas\Mvc\MvcEvent))
#16 .../omeka/index.php(21): Laminas\Mvc\Application->run()
#17 {main} {"userId":1}

```

**Comment by Daniel-KM** [comment]
Do you have the main exception message before the stacktrace?

**Comment by sinanatra** [comment]
yes: `2021-10-06T11:10:43+00:00 ERR (3): Error: Call to a member function id() on null in .../omeka/modules/Annotate/Module.php:1037`

**Comment by Daniel-KM** [comment]
Thanks, this is fixed in new version 3.3.3.6.
